framework/content/ecl-icons/README.md

```diff
@@ -2,6 +2,6 @@
 
 Icons using europa icons font.
 
-You can optionnaly change the look of the icon by adding one of the following modifiers:
--   `ecl-icon--yellowbg` adds a yellow, circle background
--   `ecl-icon--blue-dark` uses the corporate dark blue
+You can change icon color and display by using color utilities and additional modifiers:
+- **ecl-text-primary**: blue icon
```

Comment:
well, it changes the color of the text, so it is quite obvious to me, but I don't mind changing it